test/async_sign.tests.js

```diff
@@ -34,7 +34,7 @@ describe('signing a token asynchronously', function() {
     });
 
     it('should work with none algorithm where secret is set', function(done) {
```

Comment:
Ahh good catch actually I think we can remove this test because we already test this [as part of the schema here](https://github.com/auth0/node-jsonwebtoken/blob/d3b8c1d1538800f5fa9536a6419e0ec8761d193b/test/schema.tests.js#L22-L24).
https://github.com/auth0/node-jsonwebtoken/blob/d3b8c1d1538800f5fa9536a6419e0ec8761d193b/test/schema.tests.js#L22-L24